src/lib.rs

```diff
@@ -87,8 +124,16 @@ mod serial_jtag_printer {
                         }
                         conf.write_volatile(0b001);
 
+                        let mut timeout = TIMEOUT_ITERATIONS;
                         while conf.read_volatile() & 0b011 == 0b000 {
```

Comment:
What about if we don't busy-wait here? If data isn't sent it should just overwrite the FIFO, right? I guess the question is, could we fill the FIFO and not send data in regular operation (i.e USB is connected).

Reply:
Yeah this isn't possible, we need to wait for the host to receive the packet.